lib/ansible/modules/cloud/azure/azure_rm_sqldatabase_facts.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_sqldatabase_facts
+version_added: "2.8"
+short_description: Get Azure SQL Database facts.
+description:
+    - Get facts of Azure SQL Database.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    name:
+        description:
+            - The name of the database.
+    elastic_pool_name:
+        description:
+            - The name of the elastic pool.
+    tags:
+        description:
+            - Limit results by providing a list of tags. Format tags as 'key' or 'key:value'.
+
+extends_documentation_fragment:
+    - azure
```

Comment:
azure_tags?

Reply:
that would add documentation fragment supposed for main module. for facts doc is entirely different